bestla/CMakeLists.txt

```diff
@@ -46,7 +43,12 @@ endif()
 include(GNUInstallDirs)
 add_library(${PROJECT_NAME} INTERFACE)
 add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
-	
+
+if(BTLA_USE_OPENMP)
+message(STATUS "BesTLA using OpenMP")
+target_compile_definitions(${PROJECT_NAME} INTERFACE BTLA_USE_OPENMP)
```

Comment:
As we split parallelism from the bestla library, what does BTLA_USE_OPENMP mean? OMPThreading is created for easy use of UTs. It can be moved to bestla_ut.h file.

Reply:
just to determine whether to include ```omp.h```